plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -436,6 +438,12 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
         }
 	}
 
+    private boolean shouldUseLocalCheckout() {
+        boolean properLocalCheckoutDir = config.getLocalCheckoutDir() != null && config.getLocalCheckoutDir().exists();
+        boolean onlyOnePlugin = config.getIncludePlugins() != null && config.getIncludePlugins().size() == 1;
```

Comment:
I would say that if the `localCheckoutDir` is specified but `includePlugins.length != 1` we should just abort with a helpful message.